### DISCUSSION TITLE
[FEATURE]应用上商店是否支持自选版本

### DISCUSSION THREAD

**Discussion by MaRuifu** [1Panel-dev/1Panel#6957] [Ideas]
### 1Panel 版本

v1.10.20-lts

### 请描述您的需求或者改进建议

现在部署应用如果商店下拉的选项没有指定的版本, 还需要修改镜像的版本.
修改完镜像版本后,但是在应用商店-已安装里面的版本号还是之前的下拉的版本号,与真实的不对应
希望增加自定义版本号

**Comment by zhengkunwang223** [comment]
你这种情况可以考虑用本地应用实现

**Reply by gracelsy** [reply]
你好,有的时候需要指定版本，可以选择版本哪里增加一个手动输入 就像1panel新建docker那样的手动输入镜像
